src/cryptoadvance/specter/wallet.py

```diff
@@ -733,13 +730,8 @@ def createpsbt(self, addresses:[str], amounts:[float], subtract:bool=False, subt
 
         self.setlabel(self.change_address, "Change #{}".format(self.change_index))
 
-        if fee_rate > 0.0 and fee_unit == "SAT_B":
-            # bitcoin core needs us to convert sat/B to BTC/kB
-            options["feeRate"] = (fee_rate * 1000) / 1e8
-            print(options["feeRate"])
-        elif fee_rate > 0.0:
-            fee_rate = (fee_rate / 1000) * 1e8 # convert to sats for our inner calculations of fee corrrection
-            options["feeRate"] = fee_rate
+        # bitcoin core needs us to convert sat/B to BTC/kB
+        options["feeRate"] = (fee_rate * 1000) / 1e8
```

Comment:
Just to make sure we don't get floating errors again:

```suggestion
        options["feeRate"] = round((fee_rate * 1000) / 1e8, 8)
```